src/MobManager.cpp

```diff
@@ -457,24 +462,15 @@ void MobManager::deadStep(Mob *mob, time_t currTime) {
     mob->appearanceData.iHP = mob->maxHealth;
     mob->state = MobState::ROAMING;
 
-    // reset position
+    // if mob is a group leader/follower, spawn where the group is.
     if (mob->groupLeader != 0) {
-        // mob is a group leader/follower, spawn where the group is.
         if (Mobs.find(mob->groupLeader) != Mobs.end()) {
             Mob* leaderMob = Mobs[mob->groupLeader];
             mob->appearanceData.iX = leaderMob->appearanceData.iX + mob->offsetX;
             mob->appearanceData.iY = leaderMob->appearanceData.iY + mob->offsetY;
             mob->appearanceData.iZ = leaderMob->appearanceData.iZ;
-        } else {
+        } else
             std::cout << "[WARN] deadStep: mob cannot find it's leader!" << std::endl;
```

Comment:
Nitpick, but try to keep brace styling consistent; this is a particularly gross example
```suggestion
        } else {
            std::cout << "[WARN] deadStep: mob cannot find it's leader!" << std::endl;
        }
```